app/views/cancel_ruling_email.scala.html

```diff
@@ -57,10 +58,10 @@
 
         @warning(
             if(c.application.isBTI) {
-                "Cancelling this ATaR ruling cannot be reversed."
+                messages("change_case_status.cancelled.atar.ruling.warning")
             }
             else {
-                "Cancelling this liability decision cannot be reversed."
+                messages("change_case_status.cancelled.liability.decision.warning")
```

Comment:
This is a dead code as currently, it is not possible to cancel liability. Leaving it in place for now as there are talks to reintroduce liability cancellation. It was added [here](https://github.com/hmrc/tariff-classification-frontend/pull/260/files) with some removed [here](https://github.com/hmrc/tariff-classification-frontend/pull/430/files) and [here](https://github.com/hmrc/tariff-classification-frontend/pull/537/files). However, it was not completely removed as seen in this piece of code and [here](https://github.com/hmrc/tariff-classification-frontend/blob/main/app/models/Permission.scala#L222). This [spike](https://confluence.tools.tax.service.gov.uk/display/ELSE/DDCE-5590+ATaR+Classification+-+Investigate+options+for+cancelling+a+Liability+Ruling) was recently done to cover this.